go-controller/pkg/ovn/egressgw.go

```diff
@@ -1049,6 +1049,7 @@ func (oc *Controller) cleanExGwECMPRoutes() {
 						},
 					},
 				}
+				// TODO(flaviof): This may be a good candidate for DeleteOps()
```

Comment:
This doesn't have any evident relation to egress IP?

Reply:
It does not; that is why I did not fix it. But I did not want to forget about it.
But you have a valid point. I will remove it from here to make it less confusing.